Jenkinsfile

```diff
@@ -22,12 +22,6 @@ pipeline {
   }
 
   stages {
-    stage('Authorization') {
```

Comment:
We can not remove this, we need to make sure that no one can steal credentials by checking in a different Jenkinsfile

Reply:
We discussed a bit on slack and need to investigate this more...